client/src/style.css

```diff
@@ -1253,3 +1253,30 @@ sandbox header
     from {transform: rotate(0deg);}
     to {transform: rotate(359deg);}
  }
+
+
+/* Tooltip container */
+.tooltip {
```

Comment:
@airenzp this and other css classes below should be namespaced, `.sja_root_holder .tooltip`